src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitSubscriber.java

```diff
@@ -257,16 +252,58 @@ private void resubscribe() {
         }
 
         try {
-            start();
+            subscribe();
         } catch (Exception e) {
             LOGGER.error("Can not resubscribe subscriber for queue name='{}'", queue);
             healthMetrics.disable();
         }
     }
 
+    private SubscriberMonitor basicConsume() {
+        try {
+            LOGGER.info("Start listening queue name='{}', th2 pin='{}'", queue, th2Pin);
+            return connectionManager.basicConsume(queue, this::handle, this::canceled);
+        } catch (IOException e) {
+            throw new IllegalStateException("Can not subscribe to queue = " + queue, e);
+        }
+    }
+
+    private void handle(DeliveryMetadata deliveryMetadata,
+                        Delivery delivery,
+                        Confirmation confirmProcessed) throws IOException {
+        try (Timer ignored = MESSAGE_PROCESS_DURATION_SECONDS
+                .labels(th2Pin, th2Type, queue)
+                .startTimer()) {
+            MESSAGE_SIZE_SUBSCRIBE_BYTES
+                    .labels(th2Pin, th2Type, queue)
+                    .inc(delivery.getBody().length);
```

Comment:
What's up with formatting? Are yout trying to fit into 80 chars per line or smth? 😁

Reply:
Yes )